govdelivery-subscribe/error/index.html

```diff
@@ -9,15 +9,16 @@
 {%- endblock %}
 
 {% block content_main %}
-    <h1><span class="cf-icon cf-icon-error-round"></span> Not just yet!</h1>
+    <h1><span class="cf-icon cf-icon-error-round"></span> That didn't work.</h1>
     <p class="short-desc">
-        We’re still building the email signup function for <br>
-        this beta release. It will be available soon. For now:
+        We couldn’t complete your signup request. Try <br>
+        to submit your address again in a little while. For <br>
```

Comment:
I hate all the line breaks, but the way the text was laid out originally, they were supposed to stay short enough that the text doesn't break differently on mobile or desktop. I've just copied those line lengths.

I'd be happy to have a new container created for the copy that keeps it that way, but this seemed easy enough for now. It's too short a paragraph to stretch across the page; looked kind of funny.